src/main/java/com/adobe/jenkins/github_pr_comment_build/IssueCommentGHEventSubscriber.java

```diff
@@ -161,13 +162,14 @@ public void run() {
                                                             changedRepository.getRepositoryName()
                                                     }
                                             );
+                                            break topLevel;
```

Comment:
Sadly this has broken the following workflow, in which we _expect_ multiple builds to be triggered:
- we have multiple jobs which each define `triggerPRCommentBranchProperty` e.g. one with `commentBody(".*foo this please.*")` and another with `commentBody(".*bar this please.*")`
- it was previously possible to comment on a PR like this and both jobs would be triggered:
```
foo this please
bar this please
```
- however now only 1 of them is triggered (whichever one the `for (Job...` loop gets to first)

The workaround is of course to do a separate comment for each one, but when you want to trigger 5 different custom test runs, this is a 5x increase in (a) effort to initiate, and (b) the amount of email spam sent to others watching the PR.

Perhaps if instead of doing `break` it instead checked whether `job` had already been scheduled, that would be a way of avoiding duplicates without stopping multiple different jobs being scheduled from the same comment?

Reply:
We have been seeing issues for weeks with similar problems, and your comment just made me connect it to this PR. I'm downgrading the plugin on our Jenkins to verify that this is indeed the cause.

Reply:
@sparrowt please open a new issue.
As I mentioned in the comment https://github.com/jenkinsci/github-pr-comment-build-plugin/pull/41#issuecomment-1095751180,
this would break the cases where multiple legitimate jobs would be triggered by a single comment.
Unfortunately I don't have the environment to test this on.
This PR fixes the 99% case where this plugin was broken for a long time, as reported in multiple issues by multiple people.

I can propose a short-term fix by adding a checkbox that turns off one-per-comment build  (default would be on) since that's 99% of the cases.

In the future if there is more interest to see what actually causes duplicate builds maybe there would be more time to take this on

Reply:
Sorry I never got round to this - however it looks like https://github.com/jenkinsci/github-pr-comment-build-plugin/issues/49 was opened and https://github.com/jenkinsci/github-pr-comment-build-plugin/pull/48 fixes it (released in https://github.com/jenkinsci/github-pr-comment-build-plugin/releases/tag/82.v986d7874d9c5) so thank you!